docs/resources/security_monitoring_filter.md

```diff
@@ -0,0 +1,61 @@
+---
+page_title: "datadog_security_monitoring_filter Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a Datadog Security Monitoring Filter API resource. This can be used to create and manage Datadog security monitoring filters
+
+# Resource `datadog_security_monitoring_filter`
+
+Provides a Datadog Security Monitoring Filter API resource. This can be used to create and manage Datadog security monitoring filters.
+
+## Example Usage
+
+```terraform
+resource "datadog_security_monitoring_filter" "myfilter" {
+  name = "My filter"
+
+  query = "The filter is filtering."
+  is_enabled = true
+
+  exclusion_filter {
+    name = "first"
+    query = "exclude some logs"
+  }
+
+  exclusion_filter {
+    name = "second"
+    query = "exclude some other logs"
+  }
+  
+}
+```
+
+## Schema
+
+### Required
+
+- **is_enabled** (String, Required) The name of the security filter.
```

Comment:
Wrong name

Reply:
> LGTM beside a couple of things in the docs


Yes, maybe @DataDog/documentation would have some ideas for the main description of the ressource?

Reply:
Can someone add the docs team to the repo so they can be added as a reviewer?